### ISSUE TITLE
When update is called, slide snaps to current slide

### ISSUE THREAD

**Issue by iosscripts** [iosscripts/iosslider#171]
Change to not snap back on update. This is especially important when calling the update method during dragging. This new functionality would allow for out-of-the-box slide buffering.

Ref feature request/issue #2

**Comment by JacopKane** [comment]
Any update on this?
This going to first slide on update is really annoying and modifying settings data is not fixing it.

**Comment by marcwhitbread** [comment]
Currently there is no option in the settings to enable/disable the snapping on update. So you won't be able to do this.

I will take a look at how easy this might be to add as an option to enable/disable snapping. I will update this issue if/when I decide to move forward with this feature.

What are you trying to do? Maybe I can offer you a workaround?

**Comment by JacopKane** [comment]
Yeah but i'm talking about startAtSlide attribute. I found a ugly hack few weeks ago for this:

I'm changing slider's start setting before, and after that:

``` javascript
onSliderUpdate  : function(args) {
    var startAtSlide = self.options.$slider
        .data('iosslider').settings.startAtSlide;
    $slider.iosSlider('goToSlide', startAtSlide);
}
```
